action.yml

```diff
@@ -1,16 +1,27 @@
-name: 'Github Tag Bump'
-description: 'Bump and push git tag on merge'
-author: 'Nick Sjostrom'
+name: "GitHub Tag Bump"
+description: "Automatically bump and push git tags based on commit messages"
+author: "Nick Sjostrom"
+
+inputs:
+  DEFAULT_BUMP:
+    description: "Default semver bump type (major, minor, patch, or none)"
+    required: false
+    default: "minor"
+  DEFAULT_BRANCH:
+    description: "The default branch for the repository"
+    required: false
```

Comment:
Question: why are you adding these two values as inputs when env-vars provide everything else? Keeping this way would mean, to adjust the examples to something like:
```
      - name: Bump version and push tag
        uses: anothrNick/github-tag-action@1.61.0 # Don't use @master unless you're happy to test the latest version
        env:
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
          WITH_V: true
       with:
          DEFAULT_BUMP: "patch"
```

As the inputs need to be provided in a different matter. Also accessing the values would need to change to:
```
${{ inputs.DEFAULT_BUMP }}
```

Last but not least, they should be renamed to kebab-case `default-bump`.

I would delete lines 5 - 12 in this case